libvips/foreign/webpsave.c

```diff
@@ -746,6 +759,9 @@ vips_foreign_save_webp_build(VipsObject *object)
 		return -1;
 	}
 
+	if (!vips_object_argument_isset(object, "passes"))
+		webp->passes = vips_object_argument_isset(object, "target_size") ? 3 : 1;
```

Comment:
`->passes` already defaults to `1`, so I think this could be written as:
```suggestion
	if (!vips_object_argument_isset(object, "passes") &&
		vips_object_argument_isset(object, "target_size"))
		webp->passes = 3;
```

Reply:
Yes, this sounds useful. You'll need a quick note in the docs explaining the behaviour as well.